### PR TITLE
Make the editing docs button more visible and prominent.

### PR DESCRIPTION
**Pull request by nisarhassan12**
Fixes gitpod-io/website#905

This is how it looks:

image

### CODE REVIEW

**Comment by ChristinFrohne** [comment]
Thanks!

- [ ] please align the button to the right so that its on the same height as the text?
- [ ] could you add a hover? A lighter blue like #5093EF would be great

**Comment by nisarhassan12** [comment]
> - [ ] please align the button to the right so that it's on the same height as the text?

I have aligned the button to the left with the navigation(though it is not perfectly aligned across all screen sizes). Having it on the same height as the text is doable but it is not so simple to achieve. Is it something important to do? Thanks

**Comment by ChristinFrohne** [comment]
Sorry, I think I haven't expressed myself clearly. 

I meant, aligning the button to the right so that it stops with the text. Currently, there is a small gap. Can we simply move the button a bit to the right?
Bildschirmfoto 2021-01-04 um 16 47 24

**Comment by nisarhassan12** [comment]
@ChristinFrohne This is how it looks now:

image

File: src/components/docs/EditInGitpod.tsx [hunk 1]
```diff
@@ -1,41 +1,61 @@
...
-        border-radius: 50%;
-        transition: all .2s;
-
-        &:hover,
-        &:focus {
-            background: ${colors.text};
+        &:hover {
+            rect {
+                transition: all .2s;
+                fill: #5093EF;
```
Comment (ChristinFrohne) [inline src/components/docs/EditInGitpod.tsx:14]: I adjusted the color as it was a bit too bright

**Review by ChristinFrohne — COMMENTED** [review]
I adjusted the colors for the hover effect, as it was too bright

**Comment by nisarhassan12** [comment]
> I adjusted the colors for the hover effect, as it was too bright

Great. Thanks. Can we merge this now?

**Pull request merged**